modules/Data.py

```diff
@@ -114,6 +114,11 @@ def update_conversion_rates(output_currency, json_output_enabled):
                 log.log_error("Failed to find the exchange rate for outputCurrency {0}!".format(output_currency))
                 log.log_error("Make sure that {0} is either traded on Poloniex or supported by blockchain.info: {1}"\
                               .format(output_currency, "https://blockchain.info/api/exchange_rates_api"))
+                log.log_error("Using BTC as the output currency".format(url))
+                update_conversion_rates('BTC', json_output_enabled)
```

Comment:
I actually thought calling the function again was cleaner. It means if it changes in the future we don't need to change that logic. 

Though I can change it if you prefer.